frontend/src/Components/DataTable.tsx

```diff
@@ -13,34 +13,36 @@ import {
   Thead,
   Tr,
 } from "@chakra-ui/react";
-import { WorkStationGroup } from "../Data";
+import { GPUStats, WorkStationGroup } from "../Data";
 import { useState } from "react";
 import { useForceUpdate } from "framer-motion";
+import { keepIf } from "../Utils/Utils";
+
+// default to show group, machine_name, gpu_name, isFree, brand, and memory_total
+export const COLS: Record<string, boolean> = {
```

Comment:
The abstraction here isn't great: these bools are only relevant to this file

Reply:
https://github.com/gpuctl/gpuctl/pull/221/commits/d6b6eda6d34337f03cc9a13196529f934ebfddbd